releng/test_config.yaml

```diff
@@ -41,119 +41,143 @@ jobs:
     interval: 1h
     sigOwners: [sig-cloud-provider-gcp]
     releaseBlocking: true
+    testSuite: reboot
```

Comment:
Right now, testSuite is a required field. Alternatively, we could make it an optional field:

- generate the job based on the job name if this value is not provided
- if the value is provided, generate the job based on the provided value

I'm open to implement this change if you think that would work better.

Reply:
Implemented in https://github.com/kubernetes/test-infra/pull/32690/commits/3885e1ab92ff2ff23adc01b9dd3c6b59a98127a0

I can revert if you prefer the original implementation.